README.md

```diff
@@ -16,6 +16,13 @@ ren angular-starter MY_APP_NAME
 # Navigate into the newly renamed directory
 cd MY_APP_NAME
 
+# Relies on node 14 installed, higher versions not yet supported
+nvm use
+
```

Comment:
I had my node version at 16 by default, and got about six peerDependency conflicts.  14 was the highest I could go and get a clean build.  `nvm use` means that it will take the version that's in the `.nvmrc` file.